Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -622,6 +697,38 @@ extension SPNoteListViewController {
             notesListController.object(at: indexPath) as? Note
         }
     }
+
+    open override func setEditing(_ editing: Bool, animated: Bool) {
+        // tableView could be in edit mode already if a single cell is displaying a slide out menu
+        // Check if tableView and NoteListVC have the same isEditing value
+        // If not set them to the same before continuing
+        if tableView.isEditing != isEditing {
+            tableView.setEditing(isEditing, animated: true)
+        }
```

Comment:
Q: `tableView.setEditing()` is done, again, in L710.

Do we need to run **both?**